Formula/v8@5.8.rb

```diff
@@ -0,0 +1,118 @@
+# Track Chrome stable.
+# https://omahaproxy.appspot.com/
+class V8AT58 < Formula
+  desc "Google's JavaScript engine"
+  homepage "https://github.com/v8/v8/wiki"
+  url "https://chromium.googlesource.com/chromium/tools/depot_tools.git"
```

Comment:
The issue is getting the relevant manifest file and then syncing things up as in the sync_gateway formula, which then clones everything in the cache, having nothing to do with the original url download: https://github.com/Homebrew/homebrew-core/blob/master/Formula/sync_gateway.rb#L37-L51